lib/rubocop/target_finder.rb

```diff
@@ -94,8 +94,8 @@ def find_files(base_dir, flags)
     end
 
     def wanted_dir_patterns(base_dir, exclude_pattern, flags)
-      base_dir = base_dir.gsub('/{}/', '/\{}/')
-      dirs = Dir.glob(File.join(base_dir.gsub('/*/', '/\*/').gsub('/**/', '/\**/'), '*/'), flags)
+      base_dir = base_dir.gsub(/[\\\{\}\[\]\*\?]/) { |x| "\\#{x}" }
```

Comment:
Can you update a more descriptive name for the variable `x`?

Reply:
Made a few changes. What do you think?